src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -177,7 +178,10 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
 
       const fiatAccountSchema = quote.getFiatAccountSchema()
 
-      const fiatConnectClient = await quote.getFiatConnectClient()
+      const fiatConnectClient = await getFiatConnectClient(
+        quote.getProviderId(),
+        quote.getProviderBaseUrl()
```

Comment:
When a client already exists, the `baseUrl` is unused in `getFiatConnectClient`, which feels slightly weird... not sure if there's anything to do about it though, since if it _doesn't_ exist and we don't pass the `baseUrl`, we'd have to throw.

Reply:
Should we be able to specify a client API key in `getFiatConnectClient` as well? Not sure exactly how the logic should work, but probably a good idea to be able to pass one through here.

Reply:
chatted offline. This is something we can add as a follow up once the CF starts supporting client API keys.